.nx/version-plans/version-plan-1763497137649.md

```diff
@@ -0,0 +1,5 @@
+---
+registry-sdk: minor
+---
+
+Added a `refreshAbilityVersionPolicies` endpoint to support policy discoverability in the event that the ability version was published to the Vincent Registry before the policy version was published.
```

Comment:
When/where is this endpoint called? Will we have a "refresh deps" button on the dashboard's ability site?